doc/source/setup.txt

```diff
@@ -99,13 +99,7 @@ Alternatively, they can be run from the ``tests`` directory via
 
     ``python runtests.py``
 
-The tests should finish in about a minute. If you have `psyco <http://psyco.sourceforge.net/>`_ installed, the tests can also be run with
-
-    ``python runtests.py -psyco``
-
-which will cut the running time in half.
-
-If any test fails, please send a detailed bug report to the `mpmath issue tracker <https://github.com/fredrik-johansson/mpmath/issues>`_. The tests can also be run with `py.test <http://pylib.org/>`_. This will sometimes generate more useful information in case of a failure.
+If any test fails, please send a detailed bug report to the `mpmath issue tracker <https://github.com/fredrik-johansson/mpmath/issues>`_. The tests can also be run with `py.test <https://pytest.org/>`_. This will sometimes generate more useful information in case of a failure.
```

Comment:
The documentation does not seem to reflect pytest being a requirement.

Reply:
Correct.  Shall I just mention this requirement in the docs or I should also declare dependencies with setuptools in the setup.py?

Reply:
No strong opinion either way.

Reply:
Ok, pytest dependency now documented.  I also suggest using ``py.test --pyargs mpmath`` for testing on live installation.  Probably, runtests() could be removed eventually.

Let me know if you are ok with this.